pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -57,8 +57,13 @@ export function mounting_dialog(client, block, mode, forced_options) {
         for (const opt of forced_options)
             extract_option(split_options, opt);
     const extra_options = unparse_options(split_options);
+    if (subvol) {
+        extract_option(split_options, "subvol=" + subvol.pathname);
+        extract_option(split_options, "subvolid=" + subvol.id);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-110900-7b3be2c9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)